cuda_core/cuda/core/experimental/_event.py

```diff
@@ -96,6 +113,17 @@ def close(self):
         """Destroy the event."""
         self._mnff.close()
 
+    def __isub__(self, other):
+        return NotImplemented
+
+    def __rsub__(self, other):
+        return NotImplemented
+
+    def __sub__(self, other):
+        # return self - other
```

Comment:
`# return self - other (in milliseconds)`

?

Reply:
fixed in commit 8aca468